decidim-design/app/helpers/decidim/design/announcement_helper.rb

```diff
@@ -6,25 +6,25 @@ module AnnouncementHelper
       def announcement_sections
         [
           {
-            id: "callout_class",
+            id: t("decidim.design.helpers.callout_class"),
             contents: [
               {
                 type: :text,
-                values: ["This attribute applies an status to the announcement. By default, it uses secondary color."]
+                values: [t("decidim.design.helpers.callout_description")]
               },
               {
                 type: :table,
-                options: { headings: ["Announcement", "Callout class"] },
+                options: { headings: [t("decidim.design.components.announcement.title"), t("decidim.design.helpers.callout_class")] },
                 items: announcement_table(
-                  { text: "I am an announcement", callout_class: "success" },
-                  { text: "I am an announcement", callout_class: "warning" },
-                  { text: "I am an announcement", callout_class: "alert" },
-                  { text: "I am an announcement", callout_class: "secondary" },
+                  { text: "I am an announcement", callout_class: "Success" },
+                  { text: "I am an announcement", callout_class: "Warning" },
+                  { text: "I am an announcement", callout_class: "Alert" },
+                  { text: "I am an announcement", callout_class: "Secondary" },
```

Comment:
We should not change these, as they're used for the CSS 
```suggestion
                  { text: "I am an announcement", callout_class: "success" },
                  { text: "I am an announcement", callout_class: "warning" },
                  { text: "I am an announcement", callout_class: "alert" },
                  { text: "I am an announcement", callout_class: "secondary" },
```

Reply:
This is how it looks like in this PR: 

![Screenshot of this page](https://github.com/decidim/decidim/assets/717367/3a075615-bfe1-467c-8b29-d82313a4405f)